cdap-docs/developer-manual/source/pipelines/creating-pipelines.rst

```diff
@@ -212,6 +212,74 @@ Two interfaces are available:
   underlying ``cron`` program. The details of that program will depend on the operating
   system used by the host of the CDAP Master process.
 
+With the CDAP Studio, you can also create a schedule that launches a pipeline when another pipeline
+reaches some certain statuses, and let the scheduled pipeline use properties from the triggering
+pipeline as runtime arguments. You can also create such schedule by using :ref:`HTTP request <http-restful-api-lifecycle-schedule-add>`.
+To specify how the properties from triggering Pipeline are used as runtime arguments in the scheduled Pipeline,
```

Comment:
Arguments and plugin properties from the triggering pipeline can be passed to the triggered pipeline by setting a special schedule property. Set the 'triggering.properties.mapping' property to the stringified value of the following JSON object::